tests/fast-integration/test/fixtures/commerce-mock/lastorder-function.yaml

```diff
@@ -88,7 +88,8 @@ spec:
             const response = await sendEvent("sap.kyma.custom.inapp.order.received.v1", event.data, encoding);
             console.log("In-app message sent. Publisher proxy response:", response.status);
```

Comment:
I think `response.status` already contains the result of the `POST` request to EPP. Why do we need to return a new object on line 92? Can't you just assert that the status of the response is 2XX? This part of the code is shared everywhere, so ideally I'd avoid making "too many" changes.

Reply:
No, we get 200 every time. if we get some 5xx code from the EPP, there is an exception being thrown, which only logs an error, but doesn't return a proper statusCode, so like that I am returning the error if the eventPublishing caused an exception.